api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -252,15 +252,22 @@ message RedisProxy {
   PrefixRoutes prefix_routes = 5;
 
   // Authenticate Redis client connections locally by forcing downstream clients to issue a `Redis
-  // AUTH command <https://redis.io/commands/auth>`_ with this password before enabling any other
-  // command. If an AUTH command's password matches this password, an "OK" response will be returned
-  // to the client. If the AUTH command password does not match this password, then an "ERR invalid
-  // password" error will be returned. If any other command is received before AUTH when this
+  // AUTH command <https://redis.io/commands/auth>`_ with this password or with the one in
+  // :ref:`extra_downstream_auth_passwords<envoy_v3_api_field_extensions.filters.network.redis_proxy.v3.RedisProxy.extra_downstream_auth_passwords>`
+  // before enabling any other command. If an AUTH command's password matches one of these passwords,
+  // an "OK" response will be returned to the client. If the AUTH command password does not match this password,
+  // then an "ERR invalid password" error will be returned. If any other command is received before AUTH when this
   // password is set, then a "NOAUTH Authentication required." error response will be sent to the
   // client. If an AUTH command is received when the password is not set, then an "ERR Client sent
   // AUTH, but no password is set" error will be returned.
   config.core.v3.DataSource downstream_auth_password = 6 [(udpa.annotations.sensitive) = true];
```

Comment:
Let's go ahead and deprecate this field (and mark it as such in the release notes).

Reply:
Done